### DISCUSSION TITLE
Cant update today and even a full delete and start again didnt work

### DISCUSSION THREAD

**Discussion by spasypaddy** [BigBobbas/ESP32-S3-Box3-Custom-ESPHome#42] [Q&A] [answered]
INFO ESPHome 2024.7.0
INFO Reading configuration /config/esphome/esp32-s3box-3.yaml...
INFO Updating https://github.com/gnumpi/esphome_audio@dev-next
INFO Detected timezone 'Europe/London'
WARNING GPIO45 is a strapping PIN and should only be used for I/O with care.
Attaching external pullup/down resistors to strapping pins can cause unexpected failures.
See https://esphome.io/guides/faq.html#why-am-i-getting-a-warning-about-strapping-pins
WARNING GPIO46 is a strapping PIN and should only be used for I/O with care.
Attaching external pullup/down resistors to strapping pins can cause unexpected failures.
See https://esphome.io/guides/faq.html#why-am-i-getting-a-warning-about-strapping-pins
Failed config

micro_wake_word: [source /config/esphome/esp32-s3box-3.yaml:453]
  
  'models' is a required option for [micro_wake_word].
  
  The model parameter has moved to be a list element under the models parameter.
  model: okay_nabu
  on_wake_word_detected: 
    - voice_assistant.start: 
        wake_word: !lambda |-
          return wake_word;

No idea how to fix this? Please help

**Comment by BigBobbas** [comment, marked as answer]
hi, the new version of ESPHome released today . includes changes to microwakeword and is a 'breaking change'  detailed in the release notes / change log https://esphome.io/changelog/2024.7.0 . The new format for the config can also be seen here https://esphome.io/components/micro_wake_word.html
I am working on a new config with other features and will also include the fix for this. I'm expecting to make this available later today. and will post an announcement once available. 
Many Thanks
Rob

**Comment by spasypaddy** [comment]
Amazing, thank you. Will wait for the fix

**Reply by BigBobbas** [reply]
ive just added the fix to this post if you wanted it sooner
https://github.com/BigBobbas/ESP32-S3-Box3-Custom-ESPHome/discussions/43#discussion-6945220
👍

**Comment by BigBobbas** [comment]
I have marked this as closed, if you are still having an issue, please raise an issue [here](<https://github.com/BigBobbas/ESP32-S3-Box3-Custom-ESPHome/issues>)
Many Thanks for your support 🙏 
Best Wishes 
Rob 👍
